documentation/modules/running-migration-plan.adoc

```diff
@@ -36,4 +36,12 @@ Warm migration only:
 . Click a migration's *Status*, whether it failed or succeeded or is still ongoing, to view the details of the migration.
 +
 The *Migration details by VM* screen opens, displaying the start and end times of the migration, the amount of data copied, and a progress pipeline for each VM being migrated.
-. Expand an individual VM to view its steps and the elapsed time and state of each step.
\ No newline at end of file
+. Expand an individual VM to view its steps and the elapsed time and state of each step.
+
+
+[WARNING]
+====
+vMotion, including svMotion, and relocation must be disabled for VMs that are being imported to avoid data corruption.
+
+Some backup solutions explicitly prevent vMotion, storage vMotion, and relocation during backups by adding an entry to the `VPX_DISABLED_METHODS` table in vCenter.
```

Comment:
```suggestion
// Some backup solutions explicitly prevent vMotion, storage vMotion, and relocation during backups by adding an entry to the `VPX_DISABLED_METHODS` table in vCenter.
```

commenting out, in case others want this included